### ISSUE TITLE
enhancement.py调用

### ISSUE THREAD

**Issue by YoungJay0612** [haoxiangsnr/A-Convolutional-Recurrent-Neural-Network-for-Real-Time-Speech-Enhancement#15]
作者你好！有个问题请教一下。
在调用enhancement.py的时候，脚本中有如下一行代码：
enhanced_mag = model(mixture_mag_chunk).detach().cpu().unsqueeze(0)  # [1, T, F]
但是，运行后提示mixture_mag_chunk并没有定义，而且在脚本文件中也没有对该变量的计算，请问这个需要怎么计算修改？

**Comment by haoxiangsnr** [comment]
已更新项目，请重新拉去代码。
